### PR TITLE
chore: corrected CLI to not nest master config under `config`

### PR DESCRIPTION
**Pull request by corban-beaird**
## Description
With recent updates the CLI & api for master config, the new output json encloses everything in with a `config` key, causes tests that pull the config following the previous structure to fail. 

Instead, this corrects the output coming from the CLI to behave as previously without the additional `config` nesting.

## Test Plan
- Run e2e tests

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by corban-beaird** [comment]
This test should have failed in the initial run; however, it doesn't look like it it's included in the `test-e2e` workflow

File: e2e_tests/tests/conftest.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def checkpoint_storage_config(request: SubRequest) -> Dict[str, Any]:
     with logged_in_user(ADMIN_CREDENTIALS):
         output = subprocess.check_output(command, universal_newlines=True, stderr=subprocess.PIPE)
```
Comment (hamidzr) [inline e2e_tests/tests/conftest.py:178]: can we avoid causing this change instead of updating the usage?

**Comment by hamidzr** [comment]
> This test should have failed in the initial run; however, it doesn't look like it it's included in the `test-e2e` workflow

where did you run into the failure?

**Comment by corban-beaird** [comment]
> > This test should have failed in the initial run; however, it doesn't look like it it's included in the `test-e2e` workflow
> 
> where did you run into the failure?

The test failure occured in a e2e_gpu E2E GPU job on branch main.
https://circleci.com/gh/determined-ai/determined/1610273

File: harness/determined/cli/master.py [hunk 2]
```diff
@@ -10,11 +10,11 @@
 
 @authentication.required
 def config(args: Namespace) -> None:
-    resp = bindings.get_GetMasterConfig(cli.setup_session(args))
+    resp = bindings.get_GetMasterConfig(cli.setup_session(args)).config
```
Comment (hamidzr) [inline harness/determined/cli/master.py:13]: nit: config

**Review by hamidzr — APPROVED** [review]
🚀 let's see if we can run the test that previously failed with the new changes as well.

**Comment by corban-beaird** [comment]
Merging this in with failing check rebaseable, will resolve conflicts shortly

**Comment by corban-beaird** [comment]
Confirmed the test looked good locally, so went ahead and merged it in; ran a rebase of EE and waiting for the CI to return to resolve the merge conflict
https://app.circleci.com/pipelines/github/determined-ai/determined-ee?branch=main-rebase-corban

**Pull request merged**